templates/stats/index.html

```diff
@@ -12,20 +12,67 @@ <h2>{{ year|intcomma }} kr</h2>
         </div>
     </header>
     <div id="content">
-        <h2>Mest Cashflowat</h2>
-        <ol class="center">
-            {% for user in highscore %}
-            <li><a href="{% url 'user-show' user.user.username %}">{{ user.user.get_full_name }}</a> ({{ user.total_amount|intcomma }} kr)</li>
-            {% endfor %}
-        </ol>
-        <p>OBS. Räknar bara med utlägg mindre än 20 000 kr</p>
+        <h2 v-if="selected == 'Total summa'">Mest Cashflowat</h2>
+        <h2 v-if="selected == 'Antal cashflows'">Flest Cashflows</h2>
+        <div style="padding: 0 0 25px;">
+            <button style="width: 170px;" v-html="selected" v-on:click="click" class="btn-color theme-color"></button>
+        </div>
+        <div style="display: flex; justify-content: center;">
+            <table style="width: 600px;">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th>Namn</th>
+                        <th>Total summa</th>
+                        <th>Antal cashflows</th>
+                    </tr>
+                </thead>
+                <tbody v-if="selected == 'Total summa'">
+                        {% for user in highscore_amount %}
+                            <tr>
+                                <td style="width: 5%;">{{ forloop.counter }}</td>
+                                <td style="width: 45%;"><a href="{% url 'user-show' user.user.username %}">{{ user.user.get_full_name }}</a></td>
+                                <td style="width: 25%;">{{ user.total_amount|intcomma }} kr</td>
+                                <td style="width: 25%;">{{ user.receipts }} st</td>
+                            </tr>
+                        {% endfor %}
+                </tbody>
+                <tbody v-if="selected == 'Antal cashflows'">
```

Comment:
Code could be a bit more DRY (don't repeat yourself) 😅 

I can imagine you thought about it already, but the styling is repeated twice here even though it is exactly the same. 

I'm not sure how you would conditionally loop over one of two lists using Jinja2 (`{% for x in y %}`) . But I'm pretty sure you could achieve it with Vue and a `v-for` loop. But if it is unnecessarily difficult, it is probably fine as is. You would probably be storing the profile lists like committees are stored in `templates/admin/expenses/overview.html`.

Reply:
Actually, I maybe just found a pretty good solution: [Macros](https://jinja.palletsprojects.com/en/3.0.x/templates/#macros). Seems to be pretty much exactly for this use case.